### DISCUSSION TITLE
Preact怎么使用Quark design?

### DISCUSSION THREAD

**Discussion by lichangyuan1993** [hellof2e/quark-design#202] [Q&A]
问题：Preact怎么使用Quark design?

场景： 用vite+ Preact+ tailwindcss+ Quark做一个小项目

具体：结合Preactjs，应该安装quarkd+ quarked-react ? 或者说应该怎么用？

**Comment by xxxsf** [comment]
npm 安装就直接使用了，跟技术栈无关
